app/src/main/res/drawable/tab_tray_grid_item_selected_background.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <stroke
+        android:width="4dp"
+        android:color="#592ACB" />
```

Comment:
Can we use the @color/photonViolet70?

Reply:
Didn't know that was already defined. Thanks!